docs/docs/getting_started/hello_noir/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Creating A Project
```

Comment:
I actually like "Creating A Project" better as a title. It is more clear what the page is about.